patches/third_party-node-node_modules.py.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/third_party/node/node_modules.py b/third_party/node/node_modules.py
+index a6ae339fa34c5a1651d2a4ae613e399c588415ce..98d4febedea192709aeb024d4b681b37372f8b07 100755
+--- a/third_party/node/node_modules.py
++++ b/third_party/node/node_modules.py
+@@ -30,7 +30,7 @@ def PathToSvgo():
+ 
+ 
+ def PathToTerser():
+-  return _path_in_node_modules('terser', 'bin', 'terser')
++  return os_path.join('..', '..', 'brave', 'node_modules', 'terser', 'bin', 'terser')
```

Comment:
Do we need to add this to a list somewhere to revert in cr112 @mkarolin @emerick @cdesouza-chromium ?

Reply:
I added an issue for that 
https://github.com/brave/brave-browser/issues/28479